config/kinto.ini

```diff
@@ -11,20 +11,20 @@ use = egg:kinto
 # Backends.
 #
 # http://kinto.readthedocs.org/en/latest/configuration/settings.html#storage
-#
-# kinto.storage_backend = cliquet.storage.postgresql
-# kinto.storage_url = postgres://postgres:postgres@localhost/postgres
-# kinto.cache_backend = cliquet.cache.postgresql
-# kinto.cache_url = postgres://postgres:postgres@localhost/postgres
-# kinto.permission_backend = cliquet.permission.postgresql
-# kinto.permission_url = postgres://postgres:postgres@localhost/postgres
+[backends]
```

Comment:
Please remove the ``kinto.ini`` file from the version control we do not want to add since it is now generated from the template.